cfgov/unprocessed/apps/financial-well-being/css/no-js.less

```diff
@@ -0,0 +1,12 @@
+.no-js {
+  /* Show the no-js notification when Javascript is disabled. */
+  .m-notification__no-js {
+    display: block;
+    margin: 30px;
+  }
+
+  /* Hide ROUT when JavaScript is disabled. */
```

Comment:
Code comment needs to be updated to refer to `Financial Well-Being` instead of `ROUT`.